### PR TITLE
feat: drop support for Python 3.5

### PR DESCRIPTION
**Pull request by naveen521kk**
Refs: https://github.com/browniebroke/deezer-python/issues/166
Fixes: https://github.com/browniebroke/deezer-python/issues/166

BREAKING CHANGE: Depreciate python 3.5

Did I had the commit message correctly? @browniebroke

### CODE REVIEW

**Comment by browniebroke** [comment]
> Did I had the commit message correctly?

Nearly... These commmits are a bit confusing at first. Basically, the subject should be written in lowercase. The `BREAKING` part was correct thoug 👍 

Don't worry about it, I'll squash when merging.

File: requirements.in [hunk 1]
```diff
@@ -1,12 +1,12 @@
 bandit
-black; python_version > '3.5' and implementation_name != 'pypy'
+black; implementation_name != 'pypy'
 coverage
 flake8
-flake8-isort; python_version > '3.5'
-isort; python_version > '3.5'
+flake8-isort;
+isort;
```
Comment (browniebroke) [inline requirements.in:6]: I think we can remove the trailing `;` on these:

```suggestion
flake8-isort
isort
```

File: requirements.in [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 pytest
 pytest-cov
 pytest-vcr
-pyupgrade; python_version > '3.5'
+pyupgrade;
```
Comment (browniebroke) [inline requirements.in:10]:
```suggestion
pyupgrade
```

**Review by browniebroke — COMMENTED** [review]
Nice one!

**Comment by browniebroke** [comment]
Thanks again! 🎉

**Comment by naveen521kk** [comment]
I think this project can use [Poetry](http://python-poetry.org/)?

**Comment by browniebroke** [comment]
Yes, it could, I'm quite satisfied with the current setup. Did you you find annoying to work with? 

Does Poetry works with Dependabot? Also not sure if it's compatible with python-semantic-release which this project use...

**Comment by naveen521kk** [comment]
> Did you you find annoying to work with?

Pretty much `requirements.in` and nothing else.

> Does Poetry works with Dependabot? 

yes, it does.

>  Also not sure if it's compatible with python-semantic-release which this project use...

I don't know about this. What do you think which will not be compatible? AFAIK, it will work. It uses semver and there is a command to bump version.
```
poetry version <version>
```
See https://python-poetry.org/docs/cli/#version

**Comment by browniebroke** [comment]
Ok didn't know that. It's something I've been meaning to try for a bit, but maybe it's time 👍

**Comment by browniebroke** [comment]
@allcontributors please add @naveen521kk for code, infra

**Comment by naveen521kk** [comment]
I will help you with setting up Poetry if you want to. Make a new issue and assign me.

**Pull request merged**